docker-compose.test.yml

```diff
@@ -38,6 +38,8 @@ services:
     # keep container alive for test commands with `docker-compose exec`
     command: tail -F anything
     environment:
+      - PYTHONDONTWRITEBYTECODE=1
```

Comment:
Why is that needed?

Reply:
because local folders are bind mounted onto the container, we want to avoid writing python bytecode (.pyc files) because the end-up on my machine and it conflicts (different os architecture between linux and mac)